contracts/CommitRevealVoting.sol

```diff
@@ -62,12 +62,12 @@ contract CommitRevealVoting {
     */
     function _commitVote(bytes32 _pollID, bytes32 _secretHash) internal
     {
-        require(commitPeriodActive(_pollID));
-        // prevent user from committing to zero node placeholder
-        require(_pollID != 0);
+        // prevent user from committing to a non-existent poll or one in the wrong state
+        require(_pollID != 0, "Not a valid poll");
```

Comment:
splitting hairs but I would say that even more specifically this is Not a valid pollID

Reply:
Will change.